app/controllers/api/storage.php

```diff
@@ -1284,13 +1288,15 @@
     ))
     ->param('bucketId', '', new UID(), 'Storage bucket unique ID. You can create a new storage bucket using the Storage service [server integration](https://appwrite.io/docs/server/storage#createBucket).')
     ->param('fileId', '', new UID(), 'File ID.')
+    // NOTE: this is only for the sdk generator and is not used in the action below.
```

Comment:
In notes, lets metion token param is used by resource (resource name, file name)